### PR TITLE
Use Long Con & Lesiurely Amble

### PR DESCRIPTION
**Pull request by Tokoeka**
Initial AWoL support - casts long Con on crates and garbage tourists at appropriate moments, and adds the meat walk to the meat mood

### CODE REVIEW

**Comment by Tokoeka** [comment]
Partially implements #973 - does not consider use of extract oil as a combat skill due to using it provoking the monster to auto-hit you, which I cannot deal with coding at the moment

**Comment by horrible-little-slime** [comment]
I'm not going to ask you to go AWOL to test this but we should test this. Or I guess just ping whatstheirface when we put it in and ask for bug reports

File: src/combat.ts [hunk 1]
```diff
@@ -289,6 +289,12 @@ export class Macro extends StrictMacro {
           have($skill`Gallapagosian Mating Call`),
         Macro.if_($monster`garbage tourist`, Macro.trySkill($skill`Gallapagosian Mating Call`))
       )
+      .externalIf(
+        get("longConMonster") !== $monster`garbage tourist` &&
```
Comment (docrostov) [inline src/combat.ts:293]: can we get the log from the dude who ran this in AWOL to make sure it used long con five times? i could see this maybe not actually working if mafia's settings don't properly clear longConMonster when the effect wears off.

File: src/extrovermectin.ts [hunk 2]
```diff
@@ -81,8 +81,10 @@ export function doingExtrovermectin(): boolean {
 export function crateStrategy(): "Sniff" | "Saber" | "Orb" | null {
   if (!doingExtrovermectin()) return null;
   if (
-    have($skill`Transcendent Olfaction`) &&
-    (property.getString("olfactedMonster") === "crate" || get("_olfactionsUsed") < 2)
+    (have($skill`Transcendent Olfaction`) &&
+      (property.getString("olfactedMonster") === "crate" || get("_olfactionsUsed") < 2)) ||
+    (have($skill`Long Con`) &&
```
Comment (docrostov) [inline src/extrovermectin.ts:86]: aw yeah we're conning the fuck outta these crates

**Review by docrostov — APPROVED** [review]
i am going to approve these changes but please note the one request above; i think we should check that the logic there worked and mafia's preferences properly changed when long con wore off. it is not a major issue tho and i'm fine with merging this.

**Comment by docrostov** [comment]
we have a log!

https://discord.com/channels/466605739838930955/864070665615179786/1020189698355896420

i am not going to look at this today but my only question is whether the long con logic actually worked and it cast long con 5x times.

File: src/combat.ts (hunk 1 above)
Comment (Tokoeka) [reply src/combat.ts:293]: Does Long Con wear off, theres nothing on the wiki page about it being limited?

File: src/extrovermectin.ts (hunk 2 above)
Comment (Tokoeka) [reply src/extrovermectin.ts:86]: Having checked in the log, it only cast it once, on garbage tourists. did 0 extrovermectin, so cast it not at all on the crates.

**Pull request merged**